tests/unittests/map/test_implementations.py

```
@@ -0,0 +1,144 @@
+                    map_fnc(err_sample_id),
+                    save=save,
+                    progress=False,
+                    skip_failures=skip_failures,
+                ):
+                    #####
+                    return_sample_ids.add(sample_id)
+
+            assert err_sample_id not in return_sample_ids
+
+            if not skip_failures:
+                assert err_ctx.excinfo.type == type(err)
```

Ruff (0.8.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)